RELEASE_NOTES.md

```diff
@@ -1,3 +1,86 @@
+## Cookstyle 6.0
+
+### Rubocop 0.80.1 Engine
+
+The RuboCop engine that powers Cookstyle has been updated from 0.75.1 to 0.80.1. This new engine includes hundreds of bug fixes and new features that will allow us to write even more advanced Cookstyle rules in the future. This release also renames many of RuboCop's built-in cops so if you have a complex rubocop.yml file that disables or enables rules you may see warnings instructing you to update your config.
+
+### 9 New Cops
+
+#### ChefDeprecations/DeprecatedWindowsVersionCheck
+
+The `ChefDeprecations/DeprecatedWindowsVersionCheck` cop detects cookbooks that use the legacy `older_than_win_2012_or_8?` helper method. Chef Infra Client no longer supports Windows releases before 8 / 2012 as those are end of life, so this check can be removed from cookbooks.
+
+#### ChefDeprecations/ChefWindowsPlatformHelper
+
+The `ChefDeprecations/ChefWindowsPlatformHelper` cop detects cookbooks that use the deprecated helper `Chef::Platform.windows?` to see if a system is running on Windows instead of using `platform?('windows')` or the new `windows?` helper.
+
+#### ChefDeprecations/LogResourceNotifications
+
+The `ChefDeprecations/LogResourceNotifications` cop detects notifications in the `log` resource. In Chef Infra Client 16 and later the log resource will no longer trigger an update, so these notifications will never fire. Often users utilize notifications from multiple resources to a single log resource as a means of aggregating notifications to only fire a single time. For instance, many resource updates in your cookbook may require a service to restart, but you only want to trigger that service to restart one time. Chef Infra Client 15.8 and later ship with a `notify_group` resource specifically for aggregating notifications in this way, which should be used instead.
```

Comment:
```suggestion
The `ChefDeprecations/LogResourceNotifications` cop detects notifications in the `log` resource. Beginning in Chef Infra Client 16, the log resource will no longer trigger an update, so these notifications will never fire. This will create problems for users who concatenate notifications from multiple resources into a single log resource so that those notifications only fire a single time. For example, many resource updates in your cookbook may require a service to restart, but a user may only want to trigger that service to restart once. Chef Infra Client 15.8 and later ships with a `notify_group` resource specifically for aggregating notifications in this way. This resource should be used instead.
```